src/jackett.js

```diff
@@ -16,7 +16,7 @@ function reformatTitleForSearching(name) {
 		: movieMatch
 		? movieMatch[0]
 		: name;
-	return fullMatch.replace(".", " ");
+	return fullMatch.replace(/\./g, " ");
```

Comment:
Good catch! Have you tested this? I'm not sure the `replace` method will do that. 
```suggestion
	return fullMatch.replaceAll(".", " ");
```

Reply:
I tested my version, yes. Not sure about replaceAll, I'm not a JS dev. Looks like replaceAll is a prototype function.

Reply:
They're both prototype functions :) If it works, I'm satisfied.